<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaneGenerationDialog</class>
 <widget class="QDialog" name="PlaneGenerationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crystal Planes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/crystalexplorer.qrc">
    <normaloff>:/images/plane_icon.png</normaloff>:/images/plane_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="millerIndexGroupBox">
     <property name="title">
      <string>Generate Plane</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="hSpinBox">
        <property name="minimum">
         <number>-99</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hLabel">
        <property name="text">
         <string>&lt;b&gt;h&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="kSpinBox">
        <property name="minimum">
         <number>-99</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="kLabel">
        <property name="text">
         <string>&lt;b&gt;k&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="lSpinBox">
        <property name="minimum">
         <number>-99</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lLabel">
        <property name="text">
         <string>&lt;b&gt;l&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="offsetLabel">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="offsetDoubleSpinBox">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>54</width>
          <height>54</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>kSpinBox</zorder>
     <zorder>kLabel</zorder>
     <zorder>lSpinBox</zorder>
     <zorder>lLabel</zorder>
     <zorder>offsetLabel</zorder>
     <zorder>offsetDoubleSpinBox</zorder>
     <zorder>hSpinBox</zorder>
     <zorder>colorLabel</zorder>
     <zorder>colorButton</zorder>
     <zorder>hLabel</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="symmetryEquivalentCheckBox">
          <property name="text">
           <string>Also add symmetry related planes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addPlaneButton">
          <property name="text">
           <string>Add Plane</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createSurfaceStructureButton">
     <property name="text">
      <string>Create periodic surface structure</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="currentPlanes"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="removeAllPlanesButton">
       <property name="text">
        <string>Remove all planes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePlaneButton">
       <property name="text">
        <string>Remove Plane</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/crystalexplorer.qrc"/>
  <include location="../icons/crystalexplorer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlaneGenerationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlaneGenerationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
